dotCMS/src/main/java/com/dotcms/rest/api/v1/vtl/FileVelocityReader.java

```diff
@@ -0,0 +1,39 @@
+package com.dotcms.rest.api.v1.vtl;
+
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.beans.Identifier;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.web.WebAPILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.fileassets.business.FileAsset;
+import com.dotmarketing.portlets.languagesmanager.model.Language;
+import com.liferay.util.StringPool;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+
+import static com.dotcms.rest.api.v1.vtl.VTLResource.VTL_PATH;
+
+public class FileVelocityReader implements VelocityReader {
```

Comment:
We need to add Javadoc to classes and methods

Reply:
Done